lib/web/apiserver.go

```diff
@@ -505,12 +505,13 @@ func (h *Handler) bindDefaultEndpoints() {
 	// Forwards traces to the configured upstream collector
 	h.POST("/webapi/traces", h.WithAuth(h.traces))
 
-	// Web sessions
-	h.POST("/webapi/sessions/web", httplib.WithCSRFProtection(h.WithLimiterHandlerFunc(h.createWebSession)))
+	// App sessions
 	h.POST("/webapi/sessions/app", h.WithAuth(h.createAppSession))
-	h.DELETE("/webapi/sessions", h.WithAuth(h.deleteSession))
```

Comment:
I wonder if we should keep the old routes around for a while to avoid breakages. Are these only ever used by the Web UI?

Reply:
I only see them being used by the Web UI. Although maybe there's some compatibility guarantee we make towards 3rd party clients?

Reply:
I'm ok with adding a "DELETE IN 12.0" as long as we don't finally get to deleting in 17.1 😂

Reply:
Added a deprecated session for the older routes. Didn't know when to put a "DELETE" but left a comment to this PR to give context.

Reply:
I think the compatibility guarantee is more in terms of binary versions than APIs, but this is more to play it safe.

Reply:
Let's put `DELETE IN 13` and let's go back and do this as soon as we cut the v12 branch next week so we don't forget again.